modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -2424,6 +2432,105 @@ public void testExecuteObjectActionWithUsePreferredLanguageForGuestsParameter()
 		_userLocalService.deleteUser(user);
 	}
 
+	@Test
+	public void testOnAfterAddGroovyObjectActionExecutesCallback()
+		throws Exception {
+
+		ReflectionTestUtil.setFieldValue(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				0, ObjectActionExecutorConstants.KEY_GROOVY),
+			"_objectScriptingExecutor", _originalObjectScriptingExecutor);
+
+		ObjectDefinition objectDefinition = null;
+
+		try {
+			objectDefinition = ObjectDefinitionTestUtil.publishObjectDefinition(
+				Collections.singletonList(
```

Comment:
@Aquiiiles Please keep the `ScriptManagementConfigurationTestUtil` .

And you don't need to `deleteObjectDefinition` inside the `finally`.

```java
	try (Closeable closeable =
			 ScriptManagementConfigurationTestUtil.saveWithCloseable(true)) {

		ObjectDefinition objectDefinition =
				ObjectDefinitionTestUtil.publishObjectDefinition(
					Collections.singletonList(
						new TextObjectFieldBuilder(
						).labelMap(
							LocalizedMapUtil.getLocalizedMap("First Name")
						).name(
							"firstName"
						).required(
							true
						).build()),
					ObjectDefinitionConstants.SCOPE_COMPANY);

		...
		
		_objectDefinitionLocalService.deleteObjectDefinition(
			objectDefinition);
	}
```